package.json

```diff
@@ -62,7 +63,7 @@
   "release": {
     "branches": [
       "+([0-9])?(.{+([0-9]),x}).x",
-      "master",
+      "main",
       "next",
       "next-major",
       {
```

Comment:
Since we're adding the `release.config.js`, we should get rid of the config here (the whole `release` object).

Reply:
Indeed 👍. But what about the `plugins` section, we'd lose it
It is not [there](https://github.com/Typeform/semantic-release-config/blob/main/release.config.js)

Reply:
We'd need to move those definitions there too.

Or, to make things simpler, we could keep `release` config here (replacing `master` for `main` - as we've already done), and avoid adding `@typeform/semantic-release-config` dependency.

I like this second version because we haven't open-sourced `@typeform/semantic-release-config` yet.

Reply:
Let's do that, yes. 
EDIT: Done ☑️